### PR TITLE
Add description to deliver rake tasks

### PR DESCRIPTION
**Pull request by bevanloon**
Add a description to `deliver:to_subscriber` and `deliver:to_test_email` rake tasks.
Refactor and extract the `test_email` into its own method.

### CODE REVIEW

File: lib/tasks/deliver.rake [hunk 1]
```diff
@@ -1,13 +1,19 @@
 namespace :deliver do
+  def test_email
+    OpenStruct.new(subject: "Test email", body: "This is a test email,")
+  end
+
+  desc "Send a test email to a subscriber by Id"
```
Comment (thomasleese) [inline lib/tasks/deliver.rake]: `Id` with a capital `I` there I think looks a bit odd.

File: lib/tasks/deliver.rake [hunk 2]
```diff
@@ -1,13 +1,19 @@
 namespace :deliver do
+  def test_email
+    OpenStruct.new(subject: "Test email", body: "This is a test email,")
```
Comment (thomasleese) [inline lib/tasks/deliver.rake]: I think there should be a `.` at the end of the string, rather than a `,`!

Comment (bevanloon) [reply lib/tasks/deliver.rake]: quite right....good spot.

File: lib/tasks/deliver.rake (hunk 1 above)
Comment (bevanloon) [reply lib/tasks/deliver.rake]: fixed

**Pull request merged**